lib/Jobs/jobExecutionPropTypes.js

```diff
@@ -9,7 +9,7 @@ const jobExecutionPropTypes = PropTypes.shape({
     dataType: PropTypes.string,
   }).isRequired,
   parentJobId: PropTypes.string,
-  subordinationType: PropTypes.string.isRequired,
+  subordinationType: PropTypes.string,
   sourcePath: PropTypes.string.isRequired,
```

Comment:
Could you please make it optional, as for data-export there won't be such a field.